meta-iot2000-example/recipes-example/mraa/mraa_%.bbappend

```diff
@@ -1,9 +1,22 @@
-#This patch will include the SIMATIC IOT2000 as a known board that is similar to Galileo Gen 2
+#This .bbappend file will set the mraa version to 1.5.1, where the SIMATIC 
+#IOT2000 board name is already included.
 #Also the missing "meta-java" from Intel caused trouble so we use "java2" here
 
-FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+PV="1.5.1"
 
-SRC_URI_append = " file://0001-Include-support-for-SIMATIC-IOT2000-platform.patch \
-		   file://0002-intel_galileo_rev_g.c-use-pincmd-to-set-OUT_HIGH-ins.patch"
+SRC_URI = "git://github.com/intel-iot-devkit/mraa.git"
+SRCREV="6f9b470d8d25e2c8ba1586cd9d707b870ab30010"
 
-RDEPENDS_${PN}-java = "java2-runtime"
\ No newline at end of file
+RDEPENDS_${PN}-java = "java2-runtime"
+
+#include node-mraa so the lib can be used by node-red
+PROVIDES = "node-mraa"
+
+#The mraa project has some trouble installing the module in the right place, so here is an workaround.
+#PYTHON_SITEPACKAGES_DIR contains "dist-packages", I really don't know why...
+
+FILES_python-${PN} = "${PYTHON_SITEPACKAGES_DIR}/../site-packages"
+do_install_append(){
+	install -d ${D}/usr/lib/python2.7/site-packages/
+	cp ${D}/usr/lib/python2.7/dist-packages/* ${D}/usr/lib/python2.7/site-packages
```

Comment:
I tried extactly this before, but unfortunatley I could't make it work because Yocto claimed some errors with the symlinks (symlink to .o file in non debug package isn't allowed)